assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -110,9 +50,77 @@ const cartTotals = {
 /**
  * Component that renders the Cart block when user has something in cart aka "full".
  */
-const Cart = () => {
-	const currency = getCurrencyFromPriceResponse( cartTotals );
-	const totalRowsConfig = getTotalRowsConfig( cartTotals );
+const Cart = ( { isShippingCalculatorEnabled, isShippingCostHidden } ) => {
```

Comment:
Is there some whitespace issue that you're fixing here too? This diff is really big. (I reviewed with `Hide whitespace changes` for now.)

Reply:
It’s mostly moving the `getTotalRowsConfig` function to inside the Cart component since it depends on attributes based to Cart, mainly `isShippingCostHidden`

Reply:
Ah that makes sense :)

Reply:
I have a slight preference for adding the param to `getTotalRowsConfig`, so its dependencies are explicit, and less impact on the code. Not a blocker, happy for this to merge as is :)

(There's some fun merging coming with this and #1741 both close to merge!)